### DISCUSSION TITLE
Feels good to be here

### DISCUSSION THREAD

**Discussion by Victoria-SwiftAlly** [community/community#136504] [New to GitHub]
Hi everyone 

I'm Victoria, a Project Manager. I want to start contributing to Project here,  by helping the project team stay on track ,deliver within scope, time and of high quality. please I need guidance on how to get started.

Thank you 😊.

**Comment by ygor-sena** [comment]
Hello @Victoria-SwiftAlly!

I think GitHub Projects is the way to go when we talk about a tool for planning and tracking work.

Please, check the links below: 
- Introduction to Projects: https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects
- Quickstart for Projects: https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/quickstart-for-projects
- Complete documentation: [Planning and tracking with Projects: build adaptable projects to track your work on GitHub](https://docs.github.com/en/issues/planning-and-tracking-with-projects)

Best regards

**Reply by Victoria-SwiftAlly** [reply]
Thankyou, ygor.

**Reply by ygor-sena** [reply]
You're welcome, @Victoria-SwiftAlly! If my answer was useful and you don't have any further questions, please select an answer and mark this discussion as resolved.

**Reply by ghostinhershell** [reply]
Hi @Victoria-SwiftAlly 👋🏾, it looks like @ygor-sena's response was the answer to your question here. Can you please mark their response as the answer? When you mark a question as an answer, GitHub will highlight the comment and replies to the comment to help visitors quickly find the answer.
![image](https://github.com/user-attachments/assets/b89e7161-c2d1-47e2-a058-242df633e6b2)

Thanks for hopping in here @ygor-sena! 🚀

**Comment by Nuttph** [comment]
Thank you so much

**Comment by ghostinhershell** [comment]
👋 Welcome to the GitHub Community on Discussions, @Victoria-SwiftAlly! Happy to have you here 🙂

To help keep our space clutter-free, we recommend introducing yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204) and checking out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776).
